py/server/deephaven/__init__.py

```diff
@@ -23,3 +23,33 @@
 from .stream.kafka import producer as kafka_producer
 from .table_factory import empty_table, time_table, merge, merge_sorted, new_table, DynamicTableWriter
 from .replay import TableReplayer
+import threading
+
+def create_thread_entry(thread_name):
```

Comment:
I think having this live in its own module makes sense. It's ok in deephaven, but maybe `deephaven_internal` is more appropriate?

Reply:
Yep, reasonable, this is about 70% straw man while we work out the specifics of what we even want it to do.